1.1-schnorr-signatures.ipynb

```diff
@@ -69,12 +70,12 @@
     "    * One of the y-coordinates is even, and the other is odd (since `SECP256K1_FIELD_SIZE` is odd).\n",
     "    * One of the y-coordinates is a quadratic residue (has a square root modulo the field size), and the other is not.\n",
     "\n",
-    "The bip-schnorr proposal constrains `k` such that the y-value of R is a quadratic residue modulo `SECP256K1_FIELD_SIZE`. This means that from the `x` co-ordinate, the verifier can unambigiously determine `y`.\n",
+    "The bip-0340 proposal constrains private key points `k` such that the y-value of R is a even. This means that from the `x` co-ordinate, the verifier can unambigiously determine `y`.\n",
     "\n",
     "* `k` and `SECP256K1_ORDER - k` have nonce points `R = (x, y)` and `R = (x, -y)` respectively.\n",
-    "* Only one will have a y-coordinate which is a quadratic residue modulo the field size. If a randomly generated nonce `k` does not yield a valid nonce point `R`, then the signer can negate `k` to obtain a valid nonce.\n",
-    "    \n",
-    "Whether a scalar is a quadratic residue modulo the secp256k1 field size can be determined by its [jacobi symbol](http://en.wikipedia.org/wiki/Jacobi_symbol). The Test Framework provides a `jacobi_symbol()` function. If `jacobi_symbol(y, SECP256K1_FIELD_SIZE) == 1`, then `y` is a quadratic residue."
+    "* Only one will have a y-coordinate which is a even. If a randomly generated nonce `k` does not yield a valid nonce point `R`, then the signer can negate `k` to obtain a valid nonce.\n",
```

Comment:
ebb8f52e06bdd6e77fac5c0d3694f8a4414f2442

I'm not sure if "even" is more clear than just saying quadratic residue, perhaps popping in a [wiki link](https://en.wikipedia.org/wiki/Quadratic_residue#)? The concept of even is helpful but would be an additive group thing only right?

Reply:
Yeah so the spec has changed since this workshop was originally written. Quadratic residue used to be used but now evenness is used

Reply:
ahhh ok i see